src/backend/distributed/Makefile

```diff
@@ -81,19 +88,27 @@ endif
 
 .PHONY: clean-full install install-downgrades install-all
 
+clean:
+	$(MAKE) DECODER=pgoutput -C cdc clean
+	$(MAKE) DECODER=wal2json -C cdc clean
+
 cleanup-before-install:
 	rm -f $(DESTDIR)$(datadir)/$(datamoduledir)/citus.control
 	rm -f $(DESTDIR)$(datadir)/$(datamoduledir)/citus--*
 
 install: cleanup-before-install
+	$(MAKE) DECODER=pgoutput -C cdc install
```

Comment:
This seems to be replacing `install` steps and skipping part of citus installation. Seems like we should make it a separate target.